### ISSUE TITLE
HW2 Q&A today

### ISSUE THREAD

**Issue by johnyangk** [swsnu/bd2017#12]
* It's fine if two systems' outputs are the same (= both work correctly)
* Execute applications 4 times in total (2 apps * 2 systems)
* Writing just 1 class for the app is okay
* Make sure /bin/java can be executed by the Driver(Vortex Master) process when executing vortex_app
* It's safe to start with the provided skeleton code(tar), since online documentations usually assume you're using Google Cloud and not Vortex/Spark
* Use these tools for writing the report: Web UI(Spark), IR Visualizer/Log(Vortex)
* Be creative with the apps (use other java libraries)

**Comment by johnyangk** [comment]
* The 2 apps should be different, since you're using at least 2 datasets anyways

**Comment by DopingDeveloper** [comment]
Dataset이 10MB가 약간 안되는데 (9.5MB) 사용 가능할까요?

**Comment by johnyangk** [comment]
네 괜찮습니다. 기준이 애매해질 수 있어서 데이터셋 사이즈는 9.5MB 이상만 가능하게 하겠습니다. 여기서 더 내리지 않겠습니다. 감사합니다.

**Comment by johnyangk** [comment]
Regarding data/code submission...

* Please upload the code without `vortex_runtime` and `spark_runtime`
* Don't upload datasets, and instead include `wget url/to/dataset` in the app run scripts 

This is to avoid hitting Github's max size limit.
